browse/templates/abs/404.html

```diff
@@ -0,0 +1,88 @@
+{% extends "base.html" %}
+
+{%- block title -%}
+  {%- if reason -%}
+    {%- if (reason=='not_found' or reason=='version_not_found') and arxiv_id -%}
+[{{arxiv_id}}] Article does not exist
```

Comment:
If the reason for the 404 is not found -- can we say 'Article not found'? Is there ever a case in which our code might be in error and the article in question does exist but the path is wrong? A status of 'lost' is probably a lot less anxiety-inducing for an author than 'nonexistent', from a user-friendliness perspective.

Reply:
Good idea! I've replaced the "does not exist" language with "not found" in the `400.html` template.